docs/api/tray.md

```diff
@@ -216,6 +216,8 @@ Returns `String` - the title displayed next to the tray icon in the status bar
 
 Sets when the tray's icon background becomes highlighted (in blue).
 
+**[Deprecated](https://github.com/electron/electron/blob/6-0-x/docs/api/breaking-changes.md#removal-of-deprecated-sethighlightmode-method-on-tray): will be removed in v7 without replacement.**
```

Comment:
Let's instead say:
```
**[Deprecated](/link/to/tray/header/in/docs/api/breaking-changes.md)**
```

for cleanliness here and consistency with our other Deprecation notices in docs.

Reply:
otherwise LGTM, i'll ✅ as soon as that's updated.

Reply:
all right, although i've also encountered [this variations](https://github.com/electron/electron/blob/master/docs/api/download-item.md#downloaditemgetsavepath)